assets/js/components/Table/Table.jsx

```diff
@@ -43,6 +44,59 @@ const Table = ({ config, data = [] }) => {
   const [filters, setFilters] = useState([]);
   const [currentPage, setCurrentPage] = useState(1);
 
+  const searchParamsEnabled = searchParams && setSearchParams;
+
+  const columnFiltersBindToParams = columns.filter(
+    (c) => c.filter && c.filterFromParams
+  );
+
+  useEffect(() => {
+    if (!searchParamsEnabled) return;
+    const currentParams = searchParams;
+    const filtersBindedToQs = filters.reduce((acc, curr) => {
+      const filterBoundToQs = columnFiltersBindToParams.find(
```

Comment:
Lots of stuff named Bound and Bind and James Bound here, plenty of room for renaming

Reply:
Agent zero zero bound, mission complete 🤣